brands/amenity/restaurant.json

```diff
@@ -213,6 +213,17 @@
       "name": "Bonanza Steakhouse"
     }
   },
+  "amenity/restaurant|Bonchon Chicken": {
+    "countryCodes": ["bh", "kh", "kr", "kw", "my", "ph", "sg", "th", "us"],
+    "tags": {
+      "amenity": "restaurant",
+      "brand": "Bonchon Chicken",
+      "brand:wikidata": "Q4941248",
+      "brand:wikipedia": "en:Bonchon Chicken",
+      "cuisine": "korean, sushi, fried chicken",
```

Comment:
For OpenStreetMap, the `cuisine` value needs to be semicolon delimited, and should ideally appear in order of importance.  So probably like "chicken;korean;sushi"

Reply:
Fixed!